src/test/java/de/sldk/mc/metrics/WorldSizeTest.java

```diff
@@ -0,0 +1,111 @@
+package de.sldk.mc.metrics;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+import io.prometheus.client.CollectorRegistry;
+import java.io.*;
+import java.nio.file.*;
+import java.util.Random;
+import net.bytebuddy.utility.RandomString;
+import org.bukkit.World;
+import org.bukkit.plugin.Plugin;
+import org.junit.jupiter.api.*;
+
+public class WorldSizeTest {
+
+    private static final String METRIC_NAME = "mc_world_size";
+    private static final String[] METRIC_LABELS = { "world" };
+
+    private WorldSize worldSizeMetric;
+
+    private Path path;
+    private File file;
+    private World world;
+
+    @BeforeEach
+    public void beforeEach() throws IOException{
+        worldSizeMetric = new WorldSize(mock(Plugin.class));
+        worldSizeMetric.enable();
+    }
+
+    @AfterEach
+    public void afterEach() {
+        CollectorRegistry.defaultRegistry.clear();
+        try {
+            Files.deleteIfExists(path);
+        } catch (Throwable t) { }
```

Comment:
Here, I'd `throw` instead of an empty catch block. The test would fail in case of an Exception but I think that's ok in that case, no?

```suggestion
    public void afterEach() throws IOException {
        CollectorRegistry.defaultRegistry.clear();
        Files.deleteIfExists(path);
```